src/components/Container/Container.less

```diff
@@ -0,0 +1,31 @@
+.lucid-Container {
+	overflow: hidden;
+	border-radius: 6px;
+	margin: 0 10px 10px 10px;
+	position: relative;
+	background-color: @color-white;
+	border: 1px solid @color-borderColor;
+
+	&-header {
+		padding: 12px;
+		border-bottom: 1px solid @color-gray;
```

Comment:
If we know that the header and footer will always share the same bottom border we should probably abstract this out into a variable scoped to this component.